src/components/app/details/cicdHistory/Sidebar.tsx

```diff
@@ -76,7 +76,8 @@ const Sidebar = React.memo(({ type, filterOptions, triggerHistory, hasMore, setP
             return HISTORY_LABEL.ENVIRONMENT
         }
     }
-
+    filterOptions?.sort((a,b)=>(a.label > b.label)?1:-1)
```

Comment:
We should put this before the _filterOptions on line no 68 so that we need to do the sorting only one time